features/barclamp_rabbitmq.feature

```diff
@@ -0,0 +1,17 @@
+@rabbitmq
+Feature: Test the AMQP Messaging service
+  As an administrator
+  I want to make sure the messaging service is deployed successfully
+  In order to be available for other dependent services later
```

Comment:
Second part of the sentence can be removed. We don't run cct tests after every deployment step so this is missleading.